include/sys/nvpair.h

```diff
@@ -28,7 +28,7 @@
 
 #include <sys/types.h>
 #include <sys/time.h>
-#include <sys/errno.h>
+#include <errno.h>
```

Comment:
AFAIK some code is shared among userland and kernel, so instead of "search and replace" `#include <sys/errno.h>` -> `#include <errno.h>` we could do something like this

```c
#ifdef  _KERNEL
#include <linux/errno.h>
#else
#include <errno.h>
#endif  /* _KERNEL */
```

and it should at least compile. I think this is the wrong place to do this kind of trickery though, it would probably better to keep it in the SPL.